mycore-ocfl/src/main/java/org/mycore/ocfl/MCROCFLPersistenceTransaction.java

```diff
@@ -65,8 +65,8 @@ public MCROCFLPersistenceTransaction() {
 
     @Override
     public boolean isReady() {
-        LOGGER.debug("TRANSACTION {} READY CHECK - {}", threadId, MANAGER != null);
-        return MANAGER != null && !isActive();
+        LOGGER.debug("TRANSACTION {} READY CHECK", threadId);
+        return !isActive();
```

Comment:
These debug logs are there to show the current state to... well... _debug the current state_.
If the manager stays configurable (which as we yesterday discussed doesn't make alot of sense, it was a bad thought on my part when making the classification implementation), the manager check **MUST** stay, this **WILL** cause NPEs.